ui-lib/src/main/res/ui/onboarding/values/strings.xml

```diff
@@ -1,6 +1,8 @@
 <resources>
     <string name="app_name">zashi-ui</string>
 
+    <string name="etc">…</string>
```

Comment:
maybe we could use a general_etc string

Reply:
Good catch. It's addressed in [ff3cd0d](https://github.com/Electric-Coin-Company/zashi-android/pull/1493/commits/ff3cd0df714b1cdda6cb33269b6f386810e7567a).